manager/orchestrator/taskreaper/task_reaper.go

```diff
@@ -282,6 +282,8 @@ func (tr *TaskReaper) tick() {
 
 // Stop stops the TaskReaper and waits for the main loop to exit.
 func (tr *TaskReaper) Stop() {
+	// TODO(dperny) calling stop on the task reaper twice will cause a panic
```

Comment:
'



This might be an easy fix:

```
tr.stopOnce.Do(func() {
 close(tr.stopChan)
})
```

Reply:
yeah i know the easy fix, i just didn't want to "fix" it in an unrelated PR.